resolver/ng/init.sls

```diff
@@ -19,7 +19,7 @@
     - follow_symlinks: False
     {% endif %}
     - user: root
-    - group: root
+    - group: {{ resolver.group_name }}
```

Comment:
this should be `resolver.ng`.

Reply:
I think the only change is needed now is rename `group_name` to `group.
And then we are ready to go.